pages/partials/issueResponseModal.ejs

```diff
@@ -0,0 +1,78 @@
+<div class="modal fade" id="issueResponseModal" tabindex="-1" role="dialog" aria-labelledby="issueResponseModalLabel" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="issueResponseModalLabel">Send message to student</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <form>
+          <div class="form-group">
+            <label for="message-text" class="col-form-label">Message:</label>
+            <textarea class="form-control" id="issueResponseModalMessage"></textarea>
+          </div>
+        </form>
+        <hr>
+        <div class="card">
+          <div class="card-header">Message preview</div>
+          <div class="card-body" id="issueResponseModalPreview" style="white-space: pre-line;">
+          </div>
+        </div>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+        <a href="#" role="button" class="btn btn-primary" id="issueResponseModalSend">Send message</a>
+      </div>
+    </div>
+  </div>
+</div>
+
+<script>
+(function() {
+  var instructorName = '<%= authz_data.user.name %>';
+
+  function getMessageBody(studentMessage, instructorMessage, showPlaceholder) {
+    var body = 'Hello,\nThis is in response to an issue you filed on PrairieLearn:\n\n"' + studentMessage + '"\n\n';
+    body += instructorMessage || (showPlaceholder ? '<span class="font-italic text-muted">your message here!</span>' : '');
+    body += '\n\nSincerely,\n' + instructorName;
+    return body;
+  }
+
+  $('#issueResponseModal').on('show.bs.modal', function(event) {
+    var instructorRole = '<%= authz_data.course_role %>';
+    var button = $(event.relatedTarget);
+    var uid = button.data('uid');
+    var issueId = button.data('issue-id');
+    var studentMessage = button.data('student-message');
+    var modal = $(this);
+    modal.find('.modal-title').text('Send message to ' + uid);
+    modal.find('#recipientName').val(uid);
+
+    var sendButton = modal.find('#issueResponseModalSend');
+    var message = modal.find('#issueResponseModalMessage');
+    var preview = modal.find('#issueResponseModalPreview');
+
+    message.bind("propertychange keyup input cut paste", function() {
+      preview.html(getMessageBody(studentMessage, message.val(), true));
+    });
+
+    // Sync initial message preview
+    preview.html(getMessageBody(studentMessage, message.val(), true));
+
+    sendButton.click(function() {
+      var subject = 'PrairieLearn Issue #' + issueId;
+      var body = getMessageBody(studentMessage, message.val(), false);
+      var href = 'mailto:' + uid + '?subject=' + encodeURIComponent(subject) + '&body=' + encodeURIComponent(body);
```

Comment:
While I appreciate the simplicity of this, it would be much nicer to track replies within the issue. That is, we need issue comments :-) I think this might not be too bad, actually -- we'd need a table to store them, and we'd need to render them for instructors and students, but that would be about all. This would be very helpful when we have TAs process issues in courses with a large course staff, but the professor wants to go back later and check that something was actually resolved correctly. What do you think?

The "reply to issue" dialog could also include a few options: "close after reply (checked by default)", "award full points for this question".

Reply:
Yep, that's actually what I want in the long run! This was done solely based on the comment from @trombonekenny on #910. I think keeping everything centralized in PL and visible by everyone is for the best.

Reply:
Awesome! I saw Dave's comment about that and I agree it's a reasonable placeholder.

Reply:
Do you want to see this as part of this PR? You're right, probably not too hard to do, as long as we're not trying to do anything too fancy.

Although, this probably means we have to drag email support into PL to notify students about replies. I haven't looked at the PrairieSchedule code, how bad is that?

Reply:
Up to you whether this goes into this PR. Maybe it's best to just merge this one now and add issues in a separate branch?

Email sending is very simple: https://github.com/PrairieLearn/PrairieSchedule/blob/master/emails/emails.js

Reply:
Beautiful. Sounds like another use case for PrairieLib :)

I might disable replies in this PR and do it properly later so there's no awkward period of email replies that PL doesn't know about.

Reply:
👍